src/deploy/standalone/noobaa_rsyslog.conf

```diff
@@ -42,25 +42,6 @@ $WorkDirectory /var/lib/rsyslog
 # File to store the position in the journal
 #$IMJournalStateFile imjournal.state
 
-template(name="LogseneFormat" type="list" option.json="on") {
```

Comment:
Please update the doc in non_containerized_NSFS.md `Log and Logrotate` section

Reply:
Updated the document, `non_containerized_NSFS.md`.

Reply:
Thanks @romayalon,

I updated it again. Please review again.

Reply:
woops, no worries, thank you!